### ISSUE TITLE
Calculate the FIAT amount

### ISSUE THREAD

**Issue by TobiaszCudnik** [RiseVision/rise-react-wallet#33]
We need to use an external provider for the exchange rates

**Comment by roosmaa** [comment]
We should probably remove/hide the "Displayed FIAT currency" setting until this feature gets implemented.

**Comment by roosmaa** [comment]
CryptoCompare seems to have a free API we could use for this feature: https://min-api.cryptocompare.com/data/price?fsym=RISE&tsyms=USD,JPY,EUR ([documentation](https://min-api.cryptocompare.com/))

**Comment by bobbyong** [comment]
Hi @roosmaa @TobiaszCudnik have you tried our API at http://coingecko.com/api

It's public, open, and freely available. Its quite easy to consume with little to no worry about limits.
It's already been used by many good crypto projects. Happy to answer questions and concerns you may have :)

**Comment by TobiaszCudnik** [comment]
@bobbyong Thanks for approaching us! Coingecko seems like the best service for our needs, as it lists a lot of currencies. The only problem I see is the size of the payload (10kb) and ideally we'd apply a JSON path filter (included in the request), which in our case would be:

URL: https://api.coingecko.com/api/v3/coins/rise?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false

JSON path: `market_data.current_price`

Here's the list a returned currencies, for the reference. Is would be handy to have a switch to filter out non-fiat currencies, so we can directly re-use the list for the settings dropdown.
```
["AED","ARS","AUD","BCH","BDT","BHD","BMD","BNB","BRL","BTC","CAD","CHF","CLP","CNY","CZK","DKK","EOS","ETH","EUR","GBP","HKD","HUF","IDR","ILS","INR","JPY","KRW","KWD","LKR","LTC","MMK","MXN","MYR","NOK","NZD","PHP","PKR","PLN","RUB","SAR","SEK","SGD","THB","TRY","TWD","USD","VEF","XAG","XAU","XDR","XLM","XRP","ZAR"]
```

@roosmaa As mentioned earlier in this thread, cryptocompare.com doesnt provide the proper exchange rate.

I've created #148 for the branding, as I want to close this one. We'll make sure it gets implemented before the wallet comes out of beta.

Fixed with https://github.com/RiseVision/rise-react-wallet/commit/f752b42e81057ef03050438c3aeb6c8299866d67.
